tests/test_data/inbloombakery.com/inbloombakery.json

```diff
@@ -0,0 +1,100 @@
+{
+  "author": "Ginny Dyer",
+  "canonical_url": "https://inbloombakery.com/the-best-cinnamon-rolls-ever/",
+  "host": "inbloombakery.com",
+  "description": "These are the best cinnamon rolls EVER. They're soft and fluffy with a gooey brown sugar cinnamon filling, frosted with cream cheese frosting all while drenched in a rich, buttery brown sugar cinnamon caramel sauce on the bottom.",
+  "category": "Appetizer,Breakfast",
+  "image": "https://inbloombakery.com/wp-content/uploads/2023/08/Cinnamon-Rolls-21.jpg",
+  "ingredient_groups": [
+    {
+      "ingredients": [
+        "1 1/4 cups (300 ml) whole milk, warmed to about 110 degrees",
+        "2 1/4 tsp active dry yeast",
+        "1 tsp granulated white sugar (to bloom the yeast)",
+        "4 3/4 cups (593 g) all-purpose flour, spooned and leveled",
+        "1 1/2 tsp salt",
+        "2 tbsp (25 g) granulated white sugar",
+        "2 eggs, whisked",
+        "1 tbsp vanilla",
+        "1/2 cup (112 g) unsalted butter, very softened",
+        "1/2 cup (112 g) unsalted butter, very softened",
+        "1 cup (220 g) light brown sugar, packed",
+        "2 tsp cinnamon",
+        "1/3 cup (116 g) honey",
+        "1/2 tbsp vanilla",
+        "3 tbsp (45 ml) heavy cream",
+        "1/4 tsp salt",
+        "1/2 cup (112 g) unsalted butter, very softened",
+        "1 cup (220 g) light brown sugar, packed",
+        "2 tbsp cinnamon",
+        "1/4 tsp salt",
+        "1/4 cup (60 ml) heavy cream (for pouring in between the rolls)",
+        "6 tbsp (84 g) unsalted butter, very softened",
+        "6 oz (170 g) cream cheese, cold",
+        "3/4 cup (97 g) powdered sugar",
+        "1/2 tbsp vanilla"
+      ],
+      "purpose": null
+    }
+  ],
```

Comment:
Ah, one question here: do we need test coverage on ingredient groups when a scraper does not implement specific support for them?  I think it'd probably be OK to remove the `ingredient_groups` field from the test expectations.

Reply:
I definitely agree, right now taking them out returns an error but it's worth looking into. Still good to merge here and then could be handled in a large PR.